src/main/java/sg/ncl/MainController.java

```diff
@@ -166,6 +166,12 @@ public String about() {
         return "about";
     }
 
+    @RequestMapping("/BNVirtusage")
+    public String BNVirtusage() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)